web-registry/src/hooks/useQueryBaskets.ts

```diff
@@ -15,7 +13,7 @@ export default function useQueryBaskets(): QueryBasketsResponse | undefined {
   const [baskets, setBaskets] = useState<QueryBasketsResponse | undefined>();
 
   useEffect(() => {
-    if (!api?.queryClient || !nctBasket) return;
+    if (!api?.queryClient) return;
 
     const queryClient: QueryClientImpl = new QueryClientImpl(api.queryClient);
```

Comment:
Oh, we still have old fetching logic here.

Reply:
should we add a task to the tech backlog to tackle this?

Reply:
I think we already have one to check the whole codebase for the remaining old fetching logic.
We could reference this file there.

Reply:
referenced there